README.md

```diff
@@ -73,8 +73,11 @@ Even though it is strongly preferred to rely on ECC P521 and SHA-512 where possi
 6. EnrollZ service obtains the oIAK and oIDevID certs from the CA and calls the device's `RotateOIakCert` API to persist the oIAK and oIDevID certs on the control card.
 7. The switch verifies that the IAK pub key in oIAK cert matches the one in IAK cert and that IDevID pub key in oIDevID cert matches the one in IDevID cert.
 8. The switch stores oIAK and oIDevID certs in non-volatile memory and will present them in the TPM attestation `attestz` workflow.
-9. The switch must update its default SSL profile to rely on the owner IDevID cert instead of IDevID cert.
-   - *Note: This implies that after successful enrollment the switch must force all its gRPC servers/services (such as `attestz` and `certz`) to respect the updated SSL profile relying on oIDevID cert.*
+9. The switch must use the profile created during bootz which provided the trust bundle. This will rotate the profiles cert to be the provided Owner IDevID cert and
+   sets the Owner IAK cert.
+   - *Note: This implies that after successful enrollment the switch must force all its gRPC servers/services (such as `attestz` and `certz`) to respect the updated SSL profile relying on oIDevID cert. This should have already been set as part of bootz but this should again be forced at part of enrollment*
```

Comment:
What are the expectations here?
If a grpc-server is configured to use profile `Alice`, but only `Bob` uses oIDevID. Does it mean that enrollz is expected to update the switch config implicitly from `Alice` to `Bob`? Should this be reflected only in state, or in both config & state?
What if another NMS reverts this change?

I expressed a similar opinion in https://github.com/openconfig/public/issues/1050#issuecomment-1979628410, IMO it would be best to avoid potential conflicts and use explicit configuration whenever possible

Reply:
> the expectation after enrollz is yes ALL services should be pointed at the bootz provided profile.

Yes, but if the assumption is that grpc-servers are configured during bootz (which is a good assumption), are there any additional expectations on the enrollz/attestz part? 

IMO the correct boot config should be sent from the bootstrap server, and if it's not correct (specifying a wrong profile name, etc) - it's not up to enrollz/attestz to fix such mistakes

Reply:
that should be correct